_maps/shuttles/emergency_cruise.dmm

```diff
@@ -402,7 +408,8 @@
 /turf/closed/wall/mineral/titanium,
 /area/shuttle/escape)
 "jy" = (
-/obj/machinery/sleeper{
+/obj/machinery/sleeper/syndie/fullupgrade{
+	icon_state = "sleeper";
```

Comment:
You can't var edit the icon state on this one. It will revert to being a syndie sleeper when it moves or is interacted with. If you want this, you should subtype it with the appropriate sprites, e.g. `/obj/machinery/sleeper/centcom/fullupgrade`

Or just use the regular version, it's not that much worse. Whatever you prefer.

Reply:
fixed